core/src/config.rs

```diff
@@ -10,6 +10,17 @@ pub struct SessionConfig {
     pub ap_port: Option<u16>,
 }
 
+impl Default for SessionConfig {
+    fn default() -> Self {
+        Self {
+            user_agent: "librespot rust client".to_string(),
+            device_id: "Rust Client".to_string(),
+            proxy: None,
+            ap_port: None,
+        }
+    }
+}
+
```

Comment:
The old default impl was better imo, maybe just copy the old one? The uuid dep isn't much harm.

Reply:
Alright I'll see if I can find it!